client/src/app/shared/shared-video-playlist/video-playlist.service.ts

```diff
@@ -190,16 +193,19 @@ export class VideoPlaylistService {
                )
   }
 
-  addVideoInPlaylist (playlistId: number, body: VideoPlaylistElementCreate) {
-    const url = VideoPlaylistService.BASE_VIDEO_PLAYLIST_URL + playlistId + '/videos'
+  addVideoInPlaylist (playlist: VideoPlaylistSummary, body: VideoPlaylistElementCreate) {
+    const url = VideoPlaylistService.BASE_VIDEO_PLAYLIST_URL + playlist.id + '/videos'
 
     return this.authHttp.post<{ videoPlaylistElement: { id: number } }>(url, body)
                .pipe(
                  tap(res => {
                    const existsResult = this.videoExistsCache[body.videoId]
+
                    existsResult.push({
-                     playlistId,
                      playlistElementId: res.videoPlaylistElement.id,
+                     playlistId: playlist.id,
+                     playlistDisplayName: playlist.displayName,
```

Comment:
I don't think we need to put `playlistDisplayName` to this cache/store. It can lead to some issues when the playlist is renamed for example.

Why don't you limit the cache to IDs since we don't need more information in it?